My task ID: 41
Number of Tasks: 96

TRAINING DATA

Accepting Grid:
SET_combinations | Number of Trials | Status
GGG | 100 | 
GPR | 100 | 
GRP | 100 | 
PGR | 100 | 
PPP | 100 | 
PRG | 100 | 
RGP | 100 | 
RPG | 100 | 
RRR | 100 | 

Rejecting Grid:
SET_combinations | Number of Trials | Status
GGP | 50 | 
GGR | 50 | 
GPG | 50 | 
GPP | 50 | 
GRG | 50 | 
GRR | 50 | 
PGG | 50 | 
PGP | 50 | 
PPG | 50 | 
PPR | 50 | 
PRP | 50 | 
PRR | 50 | 
RGG | 50 | 
RGR | 50 | 
RPP | 50 | 
RPR | 50 | 
RRG | 50 | 
RRP | 50 | 

----------

TESTING DATA

Accepting Grid:
SET_combinations | Number of Trials | Status
GGG | 5 | 
GPR | 5 | 
GRP | 5 | 
PGR | 5 | 
PPP | 5 | 
PRG | 5 | 
RGP | 5 | 
RPG | 5 | 
RRR | 5 | 

Rejecting Grid:
SET_combinations | Number of Trials | Status
GGP | 5 | 
GGR | 5 | 
GPG | 5 | 
GPP | 5 | 
GRG | 5 | 
GRR | 5 | 
PGG | 5 | 
PGP | 5 | 
PPG | 5 | 
PPR | 5 | 
PRP | 5 | 
PRR | 5 | 
RGG | 5 | 
RGR | 5 | 
RPP | 5 | 
RPR | 5 | 
RRG | 5 | 
RRP | 5 | 

----------

GROK DATA

Accepting Grid:
SET_combinations | Number of Trials | Status

Rejecting Grid:
SET_combinations | Number of Trials | Status

----------

CORRUPT DATA

Accepting Grid:
SET_combinations | Number of Trials | Status

Rejecting Grid:
SET_combinations | Number of Trials | Status


Metrics after epoch 50:
train_loss: 0.9766349196434021
train_accuracy: 0.0022321429569274187
test_loss: 1.0294893980026245
test_accuracy: 0.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 100:
train_loss: 0.003810920985415578
train_accuracy: 0.9994419813156128
test_loss: 0.0019302518339827657
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 150:
train_loss: 0.000535760831553489
train_accuracy: 1.0
test_loss: 0.0006183701334521174
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 200:
train_loss: 0.00026819808408617973
train_accuracy: 1.0
test_loss: 0.0003183157241437584
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 250:
train_loss: 0.00017199823923874646
train_accuracy: 1.0
test_loss: 0.00023849980789236724
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 300:
train_loss: 0.0007513607852160931
train_accuracy: 0.9988839030265808
test_loss: 0.00034794778912328184
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 350:
train_loss: 8.70066141942516e-05
train_accuracy: 1.0
test_loss: 0.00011368253035470843
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 400:
train_loss: 5.653991684084758e-05
train_accuracy: 1.0
test_loss: 7.03719852026552e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 450:
train_loss: 4.872525096288882e-05
train_accuracy: 1.0
test_loss: 5.40618748345878e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 500:
train_loss: 3.429771459195763e-05
train_accuracy: 1.0
test_loss: 4.525087570073083e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 550:
train_loss: 2.7328247597324662e-05
train_accuracy: 1.0
test_loss: 3.207046756870113e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 600:
train_loss: 2.4384708012803458e-05
train_accuracy: 1.0
test_loss: 2.8899477911181748e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 650:
train_loss: 3.0226536182453856e-05
train_accuracy: 1.0
test_loss: 2.5250250473618507e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 700:
train_loss: 1.9871060430887155e-05
train_accuracy: 1.0
test_loss: 2.16302796616219e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 750:
train_loss: 1.2263966709724627e-05
train_accuracy: 1.0
test_loss: 1.875537600426469e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 800:
train_loss: 1.4293094864115119e-05
train_accuracy: 1.0
test_loss: 1.2835957932111342e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 850:
train_loss: 1.7417258277419023e-05
train_accuracy: 1.0
test_loss: 1.5823008652660064e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 900:
train_loss: 9.909817890729755e-06
train_accuracy: 1.0
test_loss: 1.0659902727638837e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 950:
train_loss: 7.488786195608554e-06
train_accuracy: 1.0
test_loss: 7.143908987927716e-06
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 1000:
train_loss: 1.0360813575971406e-05
train_accuracy: 1.0
test_loss: 1.4497639313049149e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


LLSUB_RANK=41: 2023-09-20 11:49:15.765123: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
LLSUB_RANK=41: 2023-09-20 11:49:19.475067: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory
LLSUB_RANK=41: 2023-09-20 11:49:19.475168: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory
LLSUB_RANK=41: 2023-09-20 11:49:19.475180: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
LLSUB_RANK=41: /state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/__init__.py:98: UserWarning: unable to load libtensorflow_io_plugins.so: unable to open file: libtensorflow_io_plugins.so, from paths: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io_plugins.so']
LLSUB_RANK=41: caused by: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io_plugins.so: undefined symbol: _ZN3tsl6StatusC1EN10tensorflow5error4CodeESt17basic_string_viewIcSt11char_traitsIcEENS_18SourceLocationImplE']
LLSUB_RANK=41:   warnings.warn(f"unable to load libtensorflow_io_plugins.so: {e}")
LLSUB_RANK=41: /state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/__init__.py:104: UserWarning: file system plugins are not loaded: unable to open file: libtensorflow_io.so, from paths: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io.so']
LLSUB_RANK=41: caused by: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io.so: undefined symbol: _ZTVN3tsl13GcsFileSystemE']
LLSUB_RANK=41:   warnings.warn(f"file system plugins are not loaded: {e}")
LLSUB_RANK=41: No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
LLSUB_RANK=41: 2023-09-20 11:50:27.055312: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
LLSUB_RANK=41: 2023-09-20 11:50:27.055345: W tensorflow/compiler/xla/stream_executor/cuda/cuda_driver.cc:265] failed call to cuInit: UNKNOWN ERROR (303)
